CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.1)
+project(fastor VERSION 0.6.3)
+
+option(BUILD_TESTING "Build the testing tree." ON)
+
+set(FASTOR_INCLUDE_DIR "${fastor_SOURCE_DIR}")
+
+add_library(Fastor INTERFACE)
+
+target_include_directories(
+    Fastor INTERFACE
+        $<BUILD_INTERFACE:${FASTOR_INCLUDE_DIR}>
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+
+if(BUILD_TESTING)
+    enable_testing()
+
+    add_subdirectory(tests)
+endif()
+
+include(GNUInstallDirs)
+install(
+    DIRECTORY "${fastor_SOURCE_DIR}/Fastor"
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}")
+install(
+    TARGETS Fastor
+    EXPORT FastorTargets)
+
+export(
+    TARGETS Fastor
+    NAMESPACE Fastor::
+    FILE "${fastor_BINARY_DIR}/FastorTargets.cmake")
+export(PACKAGE Fastor)
+install(
+    EXPORT FastorTargets
+    NAMESPACE Fastor::
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/Fastor")
+
+include(CMakePackageConfigHelpers)
+configure_package_config_file(
+    "${fastor_SOURCE_DIR}/data/FastorConfig.cmake.in"
+    "${fastor_BINARY_DIR}/FastorConfig.cmake"
+    INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/Fastor"
+    PATH_VARS CMAKE_INSTALL_INCLUDEDIR)
+write_basic_package_version_file(
+    "${fastor_BINARY_DIR}/FastorConfigVersion.cmake"
+    VERSION ${fastor_VERSION}
+    COMPATIBILITY SameMajorVersion
```

Comment:
`SameMajorVersion` is to be discussed I believe. Values are explained in, [CMake documentation](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#command:write_basic_package_version_file).

Reply:
I will go with the current devel version of Fastor 0.6.4